basic/src/test/java/org/jboss/hal/testsuite/test/runtime/WebMetricsTestCase.java

```diff
@@ -93,6 +96,23 @@ public void before() throws IOException, TimeoutException, InterruptedException
         Console.withBrowser(browser).waitUntilLoaded();
     }
 
+    /**
+     * Waits until server-one is in running state. This should be replaced by more general method in future.
+     * @throws TimeoutException
+     */
+    private void waitUntilServerOneRunning() throws TimeoutException {
```

Comment:
Please move this method to `org.jboss.hal.testsuite.cli.DomainManager` and I suggest making it parametrized, where the parameter would be `serverName`.